ob_intern/Model/ShortVideoPost.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ShortVideoPost.swift
+//  ob_intern
+//
+//  Created by Soft Liampisan on 28/6/2566 BE.
+//
+
+import Foundation
+import SwiftyJSON
+
+let mockHashtag: [[String]] = [["music", "basketball", "movies"],
```

Comment:
อันนี้ทำไมถึงเอา array ครอบ array อีกทีนะครับ จริงๆ ใช้แค่นี้รึเปล่านะ ?
```suggestion
let mockHashtag: [String] = ["music", "basketball", "movies"]
```

Reply:
เพราะว่าทุกuserมีหลายhashtagไหมคะ?

Reply:
ใช่ครับ แต่ว่าใช้แค่ array single dimension ได้นะ multi dimension น่าจะไว้ใช้งานลักษณะอื่น เก็บค่าที่มีมิติมากกว่านี้ครับ

Reply:
โอเคค่ะะ หนูนึกว่าใช้ไม่ได้ค่ะ